content/en/docs/tasks/observability/distributed-tracing/configurability/index.md

```diff
@@ -216,3 +216,34 @@ spec:
       tracing:
         max_path_tag_length: <VALUE>
 {{< /text >}}
+
+## Installation
```

Comment:
This example seems out of place at the end of the doc, and especially with title "Installation".

@douglas-reid PTAL

Reply:
@douglas-reid Would you please review. Thanks.